pandas/_testing/asserters.py

```diff
@@ -817,33 +817,33 @@ def assert_extension_array_equal(
 
 # This could be refactored to use the NDFrame.equals method
 def assert_series_equal(
-    left,
-    right,
-    check_dtype=True,
-    check_index_type="equiv",
-    check_series_type=True,
-    check_less_precise=no_default,
-    check_names=True,
-    check_exact=False,
-    check_datetimelike_compat=False,
-    check_categorical=True,
-    check_category_order=True,
-    check_freq=True,
-    check_flags=True,
-    rtol=1.0e-5,
-    atol=1.0e-8,
-    obj="Series",
+    left: Series,
+    right: Series,
+    check_dtype: Union[bool, str] = True,
```

Comment:
just bool

Reply:
on L117 we call `assert_series_equal` with `check_dtype=check_dtype` and the default value of `check_dtype` in that context is `"equiv"`. If we don't type it is as `Union[bool, str]` we get the following error:  

```python
error: Argument "check_dtype" to "assert_series_equal" has incompatible type "Union[bool, str]"; expected "bool"  [arg-type]
```